### PR TITLE
Add api update subscriber channel credentials

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

File: apps/api/src/app/subscribers/e2e/update-subscriber.e2e.ts [hunk 1]
```diff
@@ -54,4 +54,46 @@ describe('Update Subscriber - /subscribers/:subscriberId (PUT)', function () {
...
+      },
+      {
+        headers: {
+          authorization: `ApiKey ${session.apiKey}`,
+        },
+      }
+    );
+
+    const response = await axiosInstance.put(
+      `${session.serverUrl}/v1/subscribers/123`,
```
Comment (scopsy) [inline apps/api/src/app/subscribers/e2e/update-subscriber.e2e.ts:75]:
```suggestion
      `${session.serverUrl}/v1/subscribers/123/credentials`,
```

File: apps/api/src/app/subscribers/usecases/update-subscriber/update-subscriber.usecase.ts [hunk 2]
```diff
@@ -38,6 +38,22 @@ export class UpdateSubscriber {
...
+      if (command.channel.credentials != null) {
+        if (command.channel.credentials.channelId != null) {
+          updatePayload['channel.credentials.channelId'] = command.channel.credentials.channelId;
+        }
+
+        if (command.channel.credentials.accessToken != null) {
+          updatePayload['channel.credentials.accessToken'] = command.channel.credentials.accessToken;
+        }
+      }
+    }
```
Comment (scopsy) [inline apps/api/src/app/subscribers/usecases/update-subscriber/update-subscriber.usecase.ts:55]: This one should allow multiple providers on the same subscriber object. uniquely by the integration identifier, so for now only a single slack should exist

Comment (djabarovgeorge) [reply apps/api/src/app/subscribers/usecases/update-subscriber/update-subscriber.usecase.ts:55]: Resolved in 0bcabdd5b03c5d7edac3671ab2d1bd1ffa7ca814

File: apps/api/src/app/subscribers/e2e/update-subscriber.e2e.ts (hunk 1 above)
Comment (djabarovgeorge) [reply apps/api/src/app/subscribers/e2e/update-subscriber.e2e.ts:75]: Resolved in 0bcabdd5b03c5d7edac3671ab2d1bd1ffa7ca814

**Pull request merged**